quick-tabs/popup.js

```diff
@@ -357,11 +357,28 @@ $(document).ready(function() {
       var str = $("#searchbox").val();
       var result = search.executeSearch(str);
       renderTabs(result);
+
+      // store the current search string 
+      chrome.storage.sync.set({"qtLastSearchString": str});
     }
   });
 
-  drawCurrentTabs();
-
+  /**
+   * Try to fetch the last search string.
+   * If present, use it to render only matched tabs list
+   * else, render all current tabs list
+   */
+  chrome.storage.sync.get(["qtLastSearchString"], function(items) {
+    var str = items["qtLastSearchString"];
+    if (typeof str !== "undefined" && str.length > 0) {
+      $("#searchbox").val(str);
+      $("#searchbox").select();
```

Comment:
Rather than searching the DOM again, you could chain `.select()` onto the previous line.

Reply:
Thanks! Will make the change.

Reply:
Made the changes